GoogleForCreators/web-stories-wp
# Media: modify getThumbnailUrl to receive a minimum width

## Summary

Modifies getThumbnailUrl to receive a minWidth.

## Relevant Technical Choices

Ignores `thumbnail` as that is cropped. Ideally there should be a `cropped` attribute in that sizes value.

## To-do

None.

## User-facing changes

None. Improves rendering in the row-based gallery when the `rowBasedGallery` flag is on.

## Testing Instructions

Enable `rowBasedGallery` and verify that wide thumbnails are no longer pixelated.

---



Fixes #3323
